examples/fluids/vortexshedding.yaml

```diff
@@ -33,9 +33,8 @@ dm_plex_box_faces: 50,20,1
 dm_plex_filename: examples/fluids/cylinder.msh
 
 # Boundary Settings
-bc_slip_z: 36, 112, 80, 120
-bc_wall: 94, 28, 50, 72
-bc_inflow: 102
-bc_freestream: 116
-wall_comps: 37, 1, 4, 103, 3, 81, 2, 59, 5, 125 #not sure of this!
+bc_slip_z: 6
+bc_wall: 5
+bc_freestream: 1,2,3,4
+wall_comps: 1,2,3
```

Comment:
These are the "physical labels" in the mesh. The wall components are those that are constrained at the wall. 1,2,3 means all three components of velocity. If you add `4`, then it'll make it also Dirichlet temperature. You can explain this better in the docs once you have it working. BTW, I notice that your code isn't compiling yet, but not sure if you need help interpreting the compiler errors (we can chat on Zulip).